doc/interfaces/data_fetch.rst

```diff
@@ -0,0 +1,22 @@
+------------------------
+Downloading the Dataset
+------------------------
+In this tutorial, you will learn to download dataset using the terminal.
+
+First, let’s create a folder::
+
+    mkdir data_folder
+
+You can view all the datasets using the command::
+
+    dipy_fetch list
+
+Download the data in the ``data_folder``::
+
+    dipy_fetch specific_dataset --out_dir data_folder
+
+For example, download ``sherbrooke_3shell`` dataset using the command::
```

Comment:
For example, to download the ``sherbrooke_3shell`` dataset,, you will run::
    dipy_fetch sherbrooke_3shell --out_dir data_folder

If with the `mkdir` command shown a directory named `data_folder` was created, I would stick to that name even if downloading a specific dataset here., otherwise it may be confusing for beginners.

Reply:
Addressed in 851afcb